how-to-guides/time-profiling.ipynb

```diff
@@ -0,0 +1,870 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "dU37erdra1TK",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "6664885a-4188-4305-dbbf-17c105d14990"
+      },
+      "source": [
+        "# Attach basic profiler\n",
+        "basic_profiler = BasicTimeProfiler()\n",
+        "basic_profiler.attach(trainer)\n",
+        "\n",
+        "trainer.run(train_loader, max_epochs=2)"
+      ],
+      "execution_count": 8,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Training Results - Epoch[1] Avg accuracy: 0.99 Avg loss: 0.03\n",
+            "Validation Results - Epoch[1] Avg accuracy: 0.99 Avg loss: 0.04\n",
+            "Epoch 1, Time Taken : 30.75150728225708\n",
+            "Training Results - Epoch[2] Avg accuracy: 0.99 Avg loss: 0.04\n",
+            "Validation Results - Epoch[2] Avg accuracy: 0.99 Avg loss: 0.04\n",
+            "Epoch 2, Time Taken : 30.99610710144043\n",
+            "Total Time: 90.9643816947937\n"
+          ],
+          "name": "stdout"
+        },
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "State:\n",
+              "\titeration: 938\n",
+              "\tepoch: 2\n",
+              "\tepoch_length: 469\n",
+              "\tmax_epochs: 2\n",
+              "\toutput: 0.02857031859457493\n",
+              "\tbatch: <class 'list'>\n",
+              "\tmetrics: <class 'dict'>\n",
+              "\tdataloader: <class 'torch.utils.data.dataloader.DataLoader'>\n",
+              "\tseed: <class 'NoneType'>\n",
+              "\ttimes: <class 'dict'>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 8
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "1zGDVT4ACJSN"
+      },
+      "source": [
+        "We can then obtain the results dictionary via [`get_results()`](https://pytorch.org/ignite/generated/ignite.contrib.handlers.time_profilers.BasicTimeProfiler.html#ignite.contrib.handlers.time_profilers.BasicTimeProfiler.get_results) and pass it to [`print_results()`](https://pytorch.org/ignite/generated/ignite.contrib.handlers.time_profilers.BasicTimeProfiler.html#ignite.contrib.handlers.time_profilers.BasicTimeProfiler.print_results) to get a nicely formatted output which contains total, minimum, maximum, mean and the standard deviation of the time taken."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 685
+        },
+        "id": "dPxd8R82BfKZ",
+        "outputId": "b1f56e28-0b1c-4347-9545-91c3f5e045aa"
+      },
+      "source": [
+        "results = basic_profiler.get_results()\n",
+        "basic_profiler.print_results(results)"
```

Comment:
@Priyansi What do you think about adding `;` here 
```python
basic_profiler.print_results(results);
```
to suppress this long line:
![Screenshot from 2021-08-17 19-52-41](https://user-images.githubusercontent.com/30748114/129769018-041b9468-fcb2-43dc-b07f-eb62b387df8b.png)

Reply:
Makes sense, because that line doesn't make a lot of sense, formatting wise right now.